### PR TITLE
feat: Add note on how to ignore files for coverage only DOCS-548

### PR DESCRIPTION
**Pull request by prcr**
Adds a note clarifying how users can ignore files for the coverage analysis only.

Fixes https://github.com/codacy/docs/issues/1684.

### :eyes: Live preview
- https://feat-add-note-ignore-coverage-docs--docs-codacy.netlify.app/repositories-configure/ignoring-files/
- https://feat-add-note-ignore-coverage-docs--docs-codacy.netlify.app/repositories-configure/codacy-configuration-file/

### CODE REVIEW

**Comment by prcr** [comment]
@claudiacarpinteiro and @nicklem please have a look before I ask more people for reviews.

**Review by nicklem — APPROVED** [review]
LGTM :shipit:

File: docs/assets/includes/coverage-ignore.md [hunk 1]
```diff
@@ -0,0 +1 @@
+To exclude files from coverage analysis only, you must ignore them directly in the tool you're using to generate coverage reports and ensure that the reports you upload to Codacy don't include coverage information for those files.
```
Comment (claudiacarpinteiro) [inline docs/assets/includes/coverage-ignore.md:1]: Wondering if we really need the **only**... 🤔 
(in "To exclude files from coverage analysis only")

**Review by claudiacarpinteiro — APPROVED** [review]
Looks good! I left a tiny comment, but it's already good to go.

**Review by madalenamelo — APPROVED** [review]

**Pull request merged**